site/content/en/docs/Installation/upgrading.md

```diff
@@ -128,7 +128,94 @@ upgrades.
    between the Agones controller being recreated and GameServers being deleted doesn't occur, and GameServers can end up stuck in erroneous states.
 1. Start and complete your control plane upgrade(s).
 1. Start and complete your node upgrades.
-1. Scale your Fleets back up and/or recreate your GameServers. 
+1. Scale your Fleets back up and/or recreate your GameServers.
 1. Run any other tests to ensure the Agones installation is still working as expected.
 1. Close your maintenance window.
 7. Congratulations - you have now upgraded to a new version of Kubernetes! 👍
+
+{{% feature publishVersion="1.41.0" %}}
+## SDK Compatibility Guarantees
+
+The SDK compatibility contract aims to ensure smooth upgrades and reduce the need for frequent
+binary redeployments for game server users.
+What’s the point of having a compatibility guarantee?
```

Comment:
```suggestion

What’s the point of having a compatibility guarantee?
```
Just so we get a new paragraph - otherwise it stays same line.

If you want to break to a newline in markdown, it's `   ` (which your editor may trim).

There's a few formatting bits in a few places like this. Might want to `make site-server` to give it a once over to ensure it's formatted how you want.